image/templates/helm/stackrox-central/templates/01-central-02-db-security.yaml

```diff
@@ -55,7 +17,7 @@ rules:
   resources:
   - securitycontextconstraints
   resourceNames:
-  - anyuid
+  - nonroot-v2
```

Comment:
Hi @Maddosaurus. Question: From looking at the OpenShift docs it seems that `nonroot-v2` is OpenShift 4.x-only? Since we also support OpenShift 3, we would need to handle this differently depending on the OpenShift version, no?